scripts/project_completion.py

```diff
@@ -0,0 +1,166 @@
+            offset = 0
+            indent = " " * offset
+            rspace = 80 - offset
+            print_stderr(output_template.format(indent, "Project name:", project_dict['name'], rspace=rspace))
+            print_stderr(output_template.format(indent, "Project ID:", project_dict['id'], rspace=rspace))
+            print_stderr(output_template.format(indent, "Project status:", project_dict['status'], rspace=rspace))
+            for sample_dict in project_dict['samples']:
+                print_stderr("")
+                offset = 4
+                indent = " " * offset
+                rspace = 80 - offset
+                print_stderr(output_template.format(indent, "Sample ID:", sample_dict['id'], rspace=rspace))
+                print_stderr(output_template.format(indent, "Sample analysis status:", sample_dict['analysis_status'], rspace=rspace))
+                print_stderr(output_template.format(indent, "Sample coverage:", sample_dict['coverage'], rspace=rspace))
+                for libprep_dict in sample_dict['libpreps']:
+                    print_stderr("")
+                    offset = 8
+                    indent = " " * offset
+                    rspace = 80 - offset
+                    print_stderr(output_template.format(indent, "Libprep ID:", libprep_dict['id'], rspace=rspace))
+                    print_stderr(output_template.format(indent, "Libprep qc status:", libprep_dict['qc'], rspace=rspace))
+                    for seqrun_dict in libprep_dict['seqruns']:
+                        print_stderr("")
+                        offset = 12
+                        indent = " " * offset
+                        rspace = 80 - offset
+                        print_stderr(output_template.format(indent, "Seqrun ID:", seqrun_dict['id'], rspace=rspace))
+                        print_stderr(output_template.format(indent, "Seqrun alignment status:", seqrun_dict['alignment_status'], rspace=rspace))
+                        print_stderr(output_template.format(indent, "Seqrun mean auto. coverage:", seqrun_dict['coverage'], rspace=rspace))
+                        if "total_reads" in seqrun_dict:
+                            print_stderr(output_template.format(indent, "Seqrun total reads:", seqrun_dict['total_reads'], rspace=rspace))
+            print_stderr("\n")
+
+
+def flowcell_summarize(flowcells, brief=False, verbose=False):
+    projects_to_analyze = \
+            organize_projects_from_flowcell(demux_fcid_dirs=flowcells,
+                                            quiet=True, create_files=False)
+    project_summarize(projects_to_analyze, brief=brief, verbose=verbose)
+
+
+if __name__=="__main__":
+    parser = argparse.ArgumentParser()
+    verbosity = parser.add_mutually_exclusive_group()
+    verbosity.add_argument("-v", "--verbose", action="store_true",
+            help="Print extended info format")
+    verbosity.add_argument("-b", "--brief", action="store_true",
+            help="Print more succint output")
```

Comment:
Haha good catch! I think that was a genuine typo. Too many consonants in a row make my fingers confused